homeassistant/components/bmw_connected_drive/quality_scale.yaml

```
@@ -0,0 +1,71 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      This integration doesn't have any events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions: done
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
```

```suggestion
  docs-installation-parameters: todo
```
missing a 
```
{% include integrations/config_flow.md %}

{% configuration_basic %}
```
block desribing input parameters

---

While not using the "new" standard options, some documentation is available: https://www.home-assistant.io/integrations/bmw_connected_drive/#configuration
Will anyways be changed with https://github.com/home-assistant/home-assistant.io/pull/36083